bc/core/utils/messages.py

```diff
@@ -1,7 +1,13 @@
+import re
 from dataclasses import dataclass
 
 from bc.core.utils.string_utils import trunc
 
+DO_NOT_POST = re.compile(
+    "(pro hac vice|notice of appearance|certificate of disclosure|corporate disclosure|add and terminate attorneys)",
+    re.IGNORECASE,
+)
```

Comment:
I think re.VERBOSE might be nice here, but it's not a must. Wouldn't have mentioned it except for the other comment I wound up making.